components/Modals/SignUp.tsx

```diff
@@ -99,16 +99,23 @@ export default function LoginModal({ onClose, state }) {
 
         <HorizontalScrollView>
           {[1, 2, 3, 4, 5].map((i) => (
-            <Avatar
-              key={i}
-              avatar={`/assets/avatars/${i}.png`}
-              size={80}
-              onClick={() => setAvatar(i)}
-              className={clsx('inline-block h-20 w-20 rounded-full border-2', {
-                'border-white': avatar === i,
-                'border-transparent': avatar !== i,
-              })}
-            />
+            <div>
```

Comment:
You'll need a few more tweaks here:
- Move the `onClick` event to the button. Otherwise, this is a button that does not do anything
- Add an [aria-pressed](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed) state that reflects whether this button is selected
- Add an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) with something like "Select avatar 1" to let users know what happens when the button is pressed
- Remove the wrapping `div`, it is unnecessary

Did you check whether the button has any default styling applied, like a background or border? If so, you may need to remove those as well.

Reply:
Thanks, @GBKS!!! to review, I am addressing the answers here according to the changes :

- I checked onClick worked perfectly, as we are passing it in the avatar component explicitly.
- I tried aria presses false and it worked perfectly but as HorizontalScrollView is a single component and we are wrapping the avatar and button inside it. Also, there is an array map in HorizontalScroll View, so every time when I am moving from one avatar to another even if I have already selected one of the images, the narrator declares button is not selected. So I tried the aria-label.
- I used aria-label="select one of the five avatar images" with the button please LMK if this is useful.(Please note narrator declares this 5 times).
- I removed the div element.

Reply:
Thanks for addressing things, just left a few more specific comments.

Did you test with a screen reader? I just did here and while the avatars were able to get focus, I could not actually select them (meaning they would get the white outline).